packages/react/src/components/Toggle/next/Toggle-test.js

```diff
@@ -0,0 +1,131 @@
+/**
+ * Copyright IBM Corp. 2021
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import Toggle from './Toggle';
+import { mount } from 'enzyme';
```

Comment:
Since we're doing the v11 variant of toggle, could you update this test to use testing-library/react? 👀  We'll be using that going forward

Reply:
Updated! Sorry, I wasn't aware of that. Can you let me know if the test style is as expected now?

Also, I couldn't find any guidelines in the developer handbook regarding react testing. If you have a documentation somewhere, please let me know!

Reply:
It's been a WIP but @tay1orjones has been working on it! I think the latest bit is added here: https://github.com/carbon-design-system/carbon/blob/main/docs/style.md#testing and we're going to start updating tests soon 🤞